reference/constraints/Range.rst

```diff
@@ -383,6 +407,28 @@ Parameter        Description
 ``{{ value }}``  The current (invalid) value
 ===============  ==============================================================
 
+minPropertyPath
+~~~~~~~~~~~~~~~
+
+**type**: ``string``
+
+.. versionadded:: 4.4
+
+    The ``minPropertyPath`` option was introduced in Symfony 4.4.
+
+It defines the object property whose value is used as ``min`` option.
+
+For example, if you want to compare the ``$endDate`` property of some object
+with regard to the ``$startDate`` property of the same object, use
+``minPropertyPath="startDate"`` in the range constraint of ``$endDate``.
+
+.. tip::
+
+    When using this option, its value is available in error messages as the
+    ``{{ limit_path }}`` placeholder. Although it's not intended to
```

Comment:
Is this true? I mean limit_path for min and max? That mean I cannot use both combined?

Reply:
I agree this can be improved. Will update the corresponding PR.

Reply:
Thanks, please add tests for using both in combination too 👌🏻

Reply:
Already done. :)